common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2188,6 +2194,11 @@
           "Must be deterministic."
         ]
       },
+      "OPTION" : {
+        "message" : [
+          "Timestamp string in the options should be in the format of 'yyyy-MM-dd HH:mm:ss[.us][zone_id]'."
```

Comment:
The given formatter might be insufficient for valid timestamps and the `[.us][zone_id]` part differs from https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html.

Reply:
We use Cast and the allowed input string is very flexible. But in the document, we should ask people to use the most standard timestamp format.

BTW, this is not really a Java timestamp format syntax, but only for document purposes. I can remove the `[.us][zone_id]` part as most people won't specify them in time travel.